src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -599,7 +609,10 @@ impl<'tcx> Tree {
          -> Result<ContinueTraversal, TransitionError> {
             let NodeAppArgs { node, mut perm, rel_pos } = args;
 
-            let old_state = perm.or_insert(LocationState::new(node.default_initial_perm));
+            let old_state = perm.or_insert(LocationState::new(
+                node.is_default_initialized,
```

Comment:
I think this can always be `false` as it can never be reached for the root node -- the root node is immediately initialized for the entire range it can ever be used for.